sdk/synapse/Azure.Analytics.Synapse.Artifacts/tests/PipelineRunClientLiveTests.cs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using Azure.Analytics.Synapse.Artifacts;
+using Azure.Analytics.Synapse.Artifacts.Models;
+using Azure.Analytics.Synapse.Tests;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.Analytics.Synapse.Artifacts.Tests
+{
+    /// <summary>
+    /// The suite of tests for the <see cref="PipelineRunClient"/> class.
+    /// </summary>
+    /// <remarks>
+    /// These tests have a dependency on live Azure services and may incur costs for the associated
+    /// Azure subscription.
+    /// </remarks>
+    public class PipelineRunClientLiveTests : RecordedTestBase<SynapseTestEnvironment>
```

Comment:
It looks like the tests are using `RecordedTestBase` directly rather than this class.   Is this a future improvement or am I overlooking something?

Reply:
Yes, I stubbed out one test class for every untested client, since the infrastructure is all the same for each one. 

A future PR will fill them all in.